lib/menu.rb

```diff
@@ -0,0 +1,22 @@
+require 'dish'
+
+class Menu
+
+  attr_reader :dishes
+
+  def initialize
+    @dishes = [
```

Comment:
I love how you have made the option to inject dishes at creation of menu.  Maybe added more work as for the challenge we could just use a preset menu. But code is flexible.